ReText/window.py

```diff
@@ -1078,6 +1090,13 @@ def closeEvent(self, closeevent):
 				return closeevent.ignore()
 		if globalSettings.saveWindowGeometry and not self.isMaximized():
 			globalSettings.windowGeometry = self.saveGeometry()
+		if globalSettings.openLastFilesOnStartup:
+			files = []
+			for i in range(0, self.tabWidget.count()):
+				tab = self.tabWidget.widget(i)
+				files.append(tab.fileName)
+			writeListToSettings("lastFileList", files)
+			writeToSettings("lastTabIndex", self.tabWidget.indexOf(self.currentTab), int)
```

Comment:
… and `globalSettings.lastTabIndex = self.tabWidget.currentIndex()` here.

What you wrote is wrong because the third argument of `writeToSettings` is the default value, not the type.

Also, note that in my suggestion I replaced `indexOf()` with `currentIndex()` which is simplier.